routes/admin.js

```diff
@@ -156,6 +171,40 @@ router.get('/subscriptions', mustBeAdminMiddleware, function (req, res, next) {
     });
 });
 
+router.get('/subscriptions_csv', mustBeAdminOrApproverMiddleware, function (req, res, next) {
+    debug("get('/subscriptions')");
+    utils.getFromAsync(req, res, '/subscriptions?embed=1&', 200, function (err, subsResponse) {
+        if (err)
+            return next(err);
+        tmp.file(function (err, path, fd, cleanup) {
+            if (err)
+                return next(err);
+            const outStream = fs.createWriteStream(path);
+            outStream.write('Status;Application;Owners;Users;Api;Plan\n');
+            for (let i = 0; i < subsResponse.items.length; ++i) {
+                const item = subsResponse.items[i];
+                const subscLine = ((item.approved) ? 'Approved' : 'Pending')+ ';'
```

Comment:
Use string templating please

Reply:
Should this also perhaps include "trusted"? As that can be important when deciding whether to approve or not?

Reply:
I wonder if we should later on also have "approve" button for pending subscriptions so that admin/approver could approve it from this page itself

Reply:
Sounds like a nice add on later